users/clients/postgres/clients.go

```diff
@@ -30,13 +30,13 @@ func NewClientRepo(db postgres.Database) clients.ClientRepository {
 }
 
 func (repo clientRepo) Save(ctx context.Context, c clients.Client) (clients.Client, error) {
-	q := `INSERT INTO clients (id, name, tags, owner, identity, secret, metadata, created_at, updated_at, status, role)
-        VALUES (:id, :name, :tags, :owner, :identity, :secret, :metadata, :created_at, :updated_at, :status, :role)
-        RETURNING id, name, tags, identity, metadata, COALESCE(owner, '') AS owner, status, created_at, updated_at`
+	q := `INSERT INTO clients (id, name, tags, owner, identity, secret, metadata, created_at, updated_at, updated_by, status, role)
```

Comment:
No need for two cases. Just set the owner to nil if it's empty. Also, it makes sense to leave `updated_by` nullable and default to it on creation, indicating the Client has never been changed.

Reply:
The same goes for Things clients.